docs/credential-modeling.md

```diff
@@ -13,7 +13,7 @@ Once we make these decisions, we can construct a VC with a **credential subject*
 
 * `credentialSubject` with these top level elements:
     * `fhirVersion`: a string representation of the semantic FHIR version the content is represented in (e.g. `1.0.*` for DSTU2, `4.0.*` for R4, where `*` is a number, not a literal asterisk)
-    * `fhirBundle`: a FHIR `Bundle` resource of type "collection" that includes all required FHIR resources (content + identity resources)
```

Comment:
Do we not care about `type` at all or do we have suggestions on what to use?

Reply:
We want to allow for use cases that cover pure collections as well as document bundles. We could limit to those two, or recommend those two... But I don't necessarily see a reason to impose a constraint here. What do you think?

Reply:
It's not something that influences parsing much, at least on our end, so I agree there's not much impetus to be strict here. But I can see that devs choose an arbitrary value, unless they follow an explicit profile, so maybe we can give some guidance, e.g. with something like this?

> The Bundle `type` defaults to  "collection" unless a different type is more appropriate.

Reply:
Yeah. I'll add a link to http://hl7.org/fhir/bundle-definitions.html#Bundle.type and say something like "implementers should choose `collection` unless a more specific type applies". (FHIR rules don't allow for "defaults" here, since it's a required field.)